WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -305,15 +305,24 @@ + (NSString *)timeZone
 + (NSDictionary *)currentActiveApplication
 {
   FBApplication *application = [FBSession activeSession].activeApplication;
+  // Can be here if no activeApplications by XCTest framework
+  if (application == nil) {
+    return
+    @{
+      @"bundleId": @"",
```

Comment:
should it be null instead of an empty string?

Reply:
Changed to return `@{}` to show no `activeApplication` attribute is blank to show something happens and no active application in xctest